src/locales/de_DE/de_DE.json

```diff
@@ -351,6 +351,17 @@
         "subtitle": "Scans und Fotokopien werden nicht akzeptiert",
         "title": "Fotoseite des Reisepasses hochladen"
     },
+    "user_consent": {
+        "button_primary": "Accept",
+        "button_secondary": "Do not accept",
+        "prompt": {
+            "button_primary": "Yes, don’t verify me",
+            "button_secondary": "Review again",
+            "button_secondary_copy": "Review again",
+            "no_consent_detail": "If you do not accept Onfido’s privacy statements and terms of service, we will not be able to verify your identity and you will exit this step.",
+            "no_consent_title": "Are you sure?"
+        }
+    },
```

Comment:
Is the user consent German translations & the all other non-English locale files not translated because we're only supporting English for this screen's content?

Reply:
Yes, this screen is only available in English for now